eng/pipelines/compliance.yml

```diff
@@ -37,6 +37,15 @@ jobs:
           artifact: "symbols - NonRTM"
           path: $(Pipeline.Workspace)\symbols\NonRTM
 
+      # Archive Symbols
+      - task: MicroBuildArchiveSymbols@5
```

Comment:
We archive symbols for releases only https://devdiv.visualstudio.com/DevDiv/_release?definitionId=257&view=mine&_a=releases.

Reply:
Thanks. Just updated the release pipeline.